app/static/css/admin.css

```diff
@@ -47,9 +48,11 @@ a.section:link, a.section:visited {
     color: var(--link-fg);
 }
 
-.theme-toggle svg.theme-icon-when-auto, .theme-toggle svg.theme-icon-when-dark, .theme-toggle svg.theme-icon-when-light {
-    fill: var(--link-fg);
-    color: #fff;
+.theme-toggle svg.theme-icon-when-auto,
+.theme-toggle svg.theme-icon-when-dark,
+.theme-toggle svg.theme-icon-when-light {
+    fill: var(--primary-black);
+    color: var(--primary-white);
```

Comment:
I can't help but assume this shouldn't be needed. The built-in definition just references other variables. Shouldn't those just be adapted?  Either way, it looks like our light and dark has the same colours in this area, so why is an override then needed in the dark mode file?

Reply:
Okay, I will keep it in the admin.css, and remove the duplication in the dark mode file